src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -655,6 +659,15 @@ export default withOnyx<MoneyRequestViewPropsWithoutTransaction, MoneyRequestVie
                 return `${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`;
             },
         },
+        transactionBackup: {
+            key: ({report, parentReportActions}) => {
```

Comment:
We're using the same logic twice, could we abstract this away into a method similar to `getTransactionId`?
<img width="836" alt="Screenshot 2024-07-22 at 14 19 01" src="https://github.com/user-attachments/assets/0e349db9-b211-4aad-a729-0c5dcdafc5ff">

Reply:
@eVoloshchak Makes sense. I updated.